libs/blocks/featured-article/featured-article.js

```diff
@@ -1,6 +1,16 @@
 import { getMetadata, createTag, getConfig } from '../../utils/utils.js';
-// import fetchTaxonomy from '../../scripts/taxonomy.js';
-// import { updateLinkWithLangRoot } from '../../utils/helpers.js';
+import fetchTaxonomy from '../../scripts/taxonomy.js';
+import { updateLinkWithLangRoot } from '../../utils/helpers.js';
+
+function createCategoryLink(doc, categoryEl) {
+  const category = getMetadata('article:tag', doc);
+  return fetchTaxonomy(getConfig(), '/topics')
+    .then((taxonomy) => {
+      const categoryTaxonomy = taxonomy.get(category) || taxonomy.get('News');
+      const categoryLink = createTag('a', { href: updateLinkWithLangRoot(categoryTaxonomy.link) }, categoryTaxonomy.name);
```

Comment:
Is it certain that a `categoryTaxonomy` will always exist? If not, I'd use some optional chaining for `.link` and `.name` to be on the safe side.